.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "editor.formatOnSave": true,
+    "files.insertFinalNewline": true,
```

Comment:
🐼 

Not required ; such a formatting rule should be already controlled by the cross-IDE/text editor `.editorconfig` file!

https://editorconfig.org/
https://github.com/noosenergy/dash-mdc-neptune/blob/master/.editorconfig#L10

Reply:
Thanks, removed it!